### ISSUE TITLE
Isolate button styling? (decide after navbar styling)

### ISSUE THREAD

**Issue by PeregrineTEL** [PeregrineTEL/TEKDB#29]
Once restyling navbar is complete, decide if button css should be isolated.

I made the same styling to buttons on several CSS sheets. It would be nice to have all that styling in the same place. What would be the best way to go about doing this? A separate css sheet just for button styling? A sheet to collect button styling and other things I add might ad that will be applied to more than one html page? 
I don't think there is a single class to which all the buttons belong. "btn" is probably the most commonly applied. (Is there a tool that creates a "breadcrumb trail" or a tree like structure, but with classes instead of element names?)

**Comment by rhodges** [comment]
You are free to create a separate CSS sheet for just about anything. One for just buttons is not unheard of if  your rules get complex and you want ALL of these rules applied to the entire page you add the CSS to.

The best thing to do is map out the scope of the problem: Which buttons do I want to impact? Which pages have those buttons? Which pages do those pages inherit from? Drawing out this hierarchy then identifying the lowest common denominator will always work. 

Sometimes if one more node up in the hierarchy would be a good -- but not perfect -- fit you can evaluate the difference in effort (both in up-front effort and clarity of code for future maintenance): how much effort will it take to apply styles to the pages that I miss vs. how much effort will it take to overwrite these changes on pages that I don't wish it to apply to?

Hope that helps.

**Comment by PeregrineTEL** [comment]
Button styling moved to new css document, base.css
